deploy/kubernetes/install-kustomize.sh

```diff
@@ -16,6 +16,51 @@ fi
 if [ -f "kustomize" ]; then
   rm kustomize
 fi
-echo "installing latest version of kustomize"
-curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash
+
+echo "installing kustomize"
+#curl -s "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"  | bash
```

Comment:
Can we remove this commented command in order to reduce clutter in the script?

One can always look at the git history if they want to know old ways of installing.

Reply:
sure, I removed from my local and haven't pushed it yet .